app/src/forms/submission/service.js

```diff
@@ -51,6 +55,38 @@ const service = {
     return [];
   },
 
+  /**
+   * Given the data for a submission, find the file uploads.
+   *
+   * @param {object} data the data for a submission.
+   * @returns an array of file UUIDs.
+   */
+  _findFileIds: (data) => {
```

Comment:
The is a direct duplicate of what's being done in the form service. If possible, we should not be doing submissions in the form service, but that's something to figure out later.